docs/user-guide/attributes-table.md

```diff
@@ -113,10 +113,53 @@ In *Edit mode*, you can delete existing features following these steps:
 
 <img src="../img/delete-line-shape.gif" />
 
+Quick Filtering
+-------------------
+
+[MapStore](https://mapstore2.geo-solutions.it/mapstore/#/) attibute table allow you to quickly filtering the table records(features) by using the 
```

Comment:
**The** MapStore Att **r** ibute Table allow **s** you to quickly filter the table records(features) by using the ...

Reply:
Typos not corrected as suggested in previous comment